client/components/ProductView.jsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable jsx-a11y/mouse-events-have-key-events */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+
+const ProductView = (props) => {
+  ProductView.propTypes = {
+    game: PropTypes.shape({
+      images: PropTypes.arrayOf(PropTypes.string),
+    }).isRequired,
+  };
+
+  const { game } = props;
+
+  const [mainImage, setMainImage] = useState();
+  const [imageZoom, setImageZoom] = useState();
+
+  const imageMouseOver = (event) => {
+    setImageZoom({
+      transform: 'scale(1.4)',
+      transformOrigin: `${event.pageX}px ${event.pageY}px`,
+    });
+  };
+
+  return (
+    <div className="productCarousel">
+      <div className="imageBar">
+        {game.images.map((image) => (
+          <input type="image" className="image" key={image} src={image} alt="product images" style={mainImage === image ? { borderBottom: '3px solid #DA3625' } : null} onClick={(event) => { setMainImage(event.target.src); }} />
```

Comment:
We should probably figure this out as a group, but there seem to be several different reds involved in the page palette. My section uses da291c, but this line in the actual page I think is da362c (you have here da3625, so it's possible I'm inspecting the wrong element on the original page)

Reply:
They have quite a few shades of red. I used the one they had labeled primary.